build/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:3.7-alpine
+
+RUN pip install awscli
```

Comment:
Localstack provides a thin wrapper around the `aws` command line interface for using it with LocalStack.
It's already installed in the localstack Docker image. So... maybe, we don't need a custom Dockerfile at all...

Reply:
I'll take a look

Reply:
Ah, I was trying that before, but there's already set entrypoint, which should not be for us, e.g. I only need to run my setup script but not expose awscli. But that was several tries before, I'll try later again with current version to see if that image will work

Reply:
@mosyp we would need to override the entrypoint when starting the container:

```
10:23:06 ❯ docker run --rm -it --entrypoint /bin/bash localstack/localstack
bash-5.0# aws
aws           awslocal      awslocal.bat
```

Reply:
okay I checked, it does not support neither curl nor jq. Does it worth then using it as `from` and install it later, could save us several minutes

Reply:
@mosyp what do you think about my suggestion at https://github.com/delta-io/delta-rs/pull/175#discussion_r605083612?

Reply:
yeap, that makes sense and also could eliminate curl too